authui/src/authflowv2/components/wide_widget.css

```diff
@@ -0,0 +1,62 @@
+@layer components {
+  :root {
+    --wide_widget__bg-color: var(--color-surface-primary);
+    --wide_widget__width: 100%;
+    --wide_widget__max-width: 605px;
+    --wide_widget__border-radius: 0;
+    --wide_widget__box-shadow: none;
+    --wide_widget__border: none;
+    --wide_widget__bg_position: center;
+    --wide_widget__bg_repeat: no-repeat;
+    --wide_widget__bg_size: cover;
+
+    @media (min-width: theme("screens.tablet")) {
+      --wide_widget__border-radius: var(--border-radius-large);
+      --wide_widget__box-shadow: none;
+      --wide_widget__border: none;
+    }
+  }
+
+  :root.dark {
+    --wide_widget__bg-color: var(--color-surface-primary);
+    --wide_widget__bg-image: ;
+  }
+
+  .wide-widget {
+    width: var(--wide_widget__width);
+    border-radius: var(--wide_widget__border-radius);
+    border: var(--wide_widget__border);
+    box-shadow: var(--wide_widget__box-shadow);
+    background-color: var(--wide_widget__bg-color);
+    background-image: var(--wide_widget__bg-image);
+    background-repeat: var(--wide_widget__bg_repeat);
+    background-position: var(--wide_widget__bg_position);
+    background-size: var(--wide_widget__bg_size);
+    @apply flex flex-col overflow-hidden;
+  }
+
+  .wide-widget-content {
+    @apply relative;
+    @apply h-full w-full;
+    @apply px-8 py-8;
+    @apply flex flex-col;
+    @apply self-center;
+    max-width: var(--wide_widget__max-width);
```

Comment:
The only differences with `.widget` is max width. Let's just introduce a modify to that.
```css
.widget--wide {
  max-width: var(--widget_wide__max-width)
}

```